packages/forma-36-website/src/pages/components/icon/index.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Icon'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Icon'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-icon--icon-default'
+---
+
+(Icons are visual hints for users to indicate actions, statuses and so on. Is used in Buttons, Icon Buttons and so on.)
+
+```jsx
+<React.Fragment>
+  <Icon icon="Edit" />
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Colors](#colors)
+   2. [Size](#size)
+   3. [Icon type](#icon)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+- Position buttons consistently in the interface
```

Comment:
- Select an icon that accurately represents the subject
- Pair icons with text
- Position buttons consistently in the interface
- Ensure the behavior of the button is consistent in all use cases
- Consider how the button fits into the context of the screen and reduce complexity where possible